### ISSUE TITLE
Cannot use in heavily restricted k8s environments

### ISSUE THREAD

**Issue by oscarcp** [txn2/kubefwd#209]
TL;DR: I work in a heavily restricted environment that makes most of the k8s tools to fail because they need extra information. Is there any way to make kubefwd strictly port forward without doing any checks or requiring any permissions apart from portForward?

At my company, k8s environments are heavily restricted, even for the development environment, which makes this tool unable to run, I'll put the specific case:

In the development environment, I don't have any access whatsoever to absolutely any functionality from k8s except for the following:
- Access two of the namespaces
- List the pods for those two namespaces
- Port forward pods from those namespaces

**Any other action is restricted** which makes not only this tool but most of the tools unable to run properly, although some of them manage to "kind of work" like k9s.

The issue that I have with kubefwd is that it can't even start and it a tool that I'd **really** like to use.

```
INFO[10:25:27]  _          _           __             _     
INFO[10:25:27] | | ___   _| |__   ___ / _|_      ____| |    
INFO[10:25:27] | |/ / | | | '_ \ / _ \ |_\ \ /\ / / _  |    
INFO[10:25:27] |   <| |_| | |_) |  __/  _|\ V  V / (_| |    
INFO[10:25:27] |_|\_\\__,_|_.__/ \___|_|   \_/\_/ \__,_|    
INFO[10:25:27]                                              
INFO[10:25:27] Version 1.21.0                               
INFO[10:25:27] https://github.com/txn2/kubefwd              
INFO[10:25:27]                                              
INFO[10:25:27] Press [Ctrl-C] to stop forwarding.           
INFO[10:25:27] 'cat /etc/hosts' to see all host entries.    
INFO[10:25:27] Loaded hosts file /etc/hosts                 
INFO[10:25:27] HostFile management: Original hosts backup already exists at /root/hosts.original 
FATA[10:25:27] Error connecting to k8s cluster: Get "http://localhost:8080/version?timeout=32s": dial tcp [::1]:8080: connect: connection refused 
```

My question is, is there any way to make kubefwd work in such a restricted environment?

**Comment by cjimti** [comment]
@oscarcp `kubefwd` does not need any additional privileges from what you describe. `kubefwd` only needs to be able to list Services and Pods in any Namespaces provided, and permission to port-forward. So, if you can `kubectl get svc -n somenamespace`, `kubectl get pods -n somenamespace` and `kubectl port-forward pod/pod-a nnnn:nnnn  -n somenamespace` then `kubefwd` should work fine. `kubefwd` only needs administrative access on your local workstation in order to add entries to `/etc/hosts` and bind to low ports.

It looks like you are trying to connect to a k8s cluster at `http://localhost:8080`, which is the default when no context is found. I doubt this is how you access your company cluster. 

1) What operating system are you using?
2) Does `kubectl` work for you?
3) Do you have a `~/.kube/config` file?
